install/share/unbound.conf.template

```diff
@@ -0,0 +1,7 @@
+server:
+    tls-cert-bundle: $TLS_CERT_BUNDLE_PATH
+    interface: 127.0.0.55
```

Comment:
perhaps we could use 127.0.0.53 as the port is 53?

Reply:
127.0.0.53 is used already by systemd-resolved.
```
       •   Additionally, systemd-resolved provides a local DNS stub listener on the IP addresses 127.0.0.53 and 127.0.0.54 on the local loopback interface. Programs issuing DNS requests directly, bypassing any local API may
           be directed to this stub, in order to connect them to systemd-resolved. Note however that it is strongly recommended that local programs use the glibc NSS or bus APIs instead (as described above), as various
           network resolution concepts (such as link-local addressing, or LLMNR Unicode domains) cannot be mapped to the unicast DNS protocol.

```

Reply:
Our plan is to disable systemd-resolved during FreeIPA deployment for the sake of simplicity. Having up to three DNS services enabled—BIND, Unbound, and systemd-resolved—would overcomplicate the design.